rules-tests/Php74/Rector/FuncCall/ArraySpreadInsteadOfArrayMergeRector/Fixture/Fixture/flatten_explicit_array_and_variable.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\Php74\Rector\FuncCall\ArraySpreadInsteadOfArrayMergeRector\Fixture;
```

Comment:
Seems the file path contains `/Fixture/Fixture`. Could you fix it to only single `/Fixture`?

Reply:
Good catch! will fix it.
Just so that you know I think then it's a bug of the Create a Test action from the demo, below what it says after a click on it:
![image](https://user-images.githubusercontent.com/1480128/229150498-aa9c3ce5-79ae-45e0-9ebe-e81191ea6b79.png)

Reply:
@yguedidi yes, it seems bug on demo page fixture link generator

Reply:
It seems github is changing url create file generation, the following check was working to point to dir, now its not

https://github.com/rectorphp/getrector-com/blob/1089452c1a66e018aa11fd552dc7d33eb4ea9c3e/tests/GitHubMagicLink/LinkFactory/FixtureLinkFactory/Fixture/symfony_expected_link.txt#L1

The Fixture/ prefix now can be removed if already in Fixture dir

Reply:
Fixture link generation fixed at https://github.com/rectorphp/getrector-com/pull/1205